packages/server/postgres/queries/getPollsById.ts

```diff
@@ -0,0 +1,8 @@
+import {getPollsByIdQuery, IGetPollsByIdQueryResult} from './generated/getPollsByIdQuery'
```

Comment:
Rename this file to `getPollsbyId`**`s`** so it's clear that it expects an array of ids